src/java/com/google/cloud/bigquery/dwhassessment/hooks/logger/utils/IdGenerator.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright 2022-2023 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.cloud.bigquery.dwhassessment.hooks.logger.utils;
+
+import org.apache.commons.lang.RandomStringUtils;
+
+/** Unique id generator for the logger. */
+public final class IdGenerator {
+  private static final int ID_LENGTH = 10;
+
+  private static final String ALLOWED_CHARACTERS =
+      "0123456789ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz";
+
+  private IdGenerator() {}
+
+  public static String generate() {
+    return RandomStringUtils.random(ID_LENGTH, ALLOWED_CHARACTERS);
```

Comment:
This can be simplified to `RandomStringUtils.randomAlphanumeric(ID_LENGTH)`.

https://commons.apache.org/proper/commons-lang/javadocs/api-2.6/org/apache/commons/lang/RandomStringUtils.html#randomAlphanumeric(int)

Reply:
Thanks, done